src/components/editPostModal/index.js

```diff
@@ -2,19 +2,27 @@ import { useState } from "react"
 import useModal from "../../hooks/useModal"
 import './style.css'
 import Button from '../button'
+import { updatePost } from "../../service/apiClient"
 
-const EditPostModal = () => {
+const EditPostModal = ({content, id, setContent}) => {
     const { closeModal } = useModal()
     const [message, setMessage] = useState(null)
-    const [text, setText] = useState('')
+    const [text, setText] = useState(content)
+
 
     const onChange = (e) => {
         setText(e.target.value)
     }
 
-    const onSubmit = () => {
+    async function onSubmit ()  {
         setMessage('Submit button was clicked! Closing modal in 2 seconds...')
-
+        let test = await updatePost(id, text)
```

Comment:
Please use variable names that make sense, in 3 months it will be hard to know what `test` means

Reply:
Also, use `const` since you're not changing the value later